src/components/interaction-designer/resource-editors/ResourceVariantTextEditor.vue

```diff
@@ -1,300 +1,57 @@
 <template>
   <div class="resource-variant-text-editor">
-    <div
```

Comment:
Thanks for thinking about this.
To be honest, I don't have a concern about removing this; but I'm afraid I don't have enough context about expected behaviour here. So I'd like to wait for @bzabos for confirmation.